Sources/LiveViewNative/Modifiers/View Styles Modifiers/ContentShapeModifier.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  ContentShapeModifier.swift
+//  
+//
+//  Created by murtza on 24/05/2023.
+//
+
+import SwiftUI
+
+/// Sets the content shape for this view.
+///
+/// ```html
+/// <HStack modifiers={content_shape(@native, shape: :rectangle)}>
+///   <Image system-name="heart.circle"></Image>
+/// </HStack>
+///
+/// <HStack modifiers={content_shape(@native, kind: :hover_effect, shape: :rectangle)}>
+///   <Image system-name="heart.circle"></Image>
+/// </HStack>
+///
+/// <HStack modifiers={content_shape(@native, kind: [:hover_effect, :interaction], shape: :rectangle, eo_fill: true)}>
+///   <Image system-name="heart.circle"></Image>
+/// </HStack>
```

Comment:
I'm not sure these examples properly demonstrate where this modifier is useful. Maybe put the modifier on a `Button` or use a tap gesture to show how it changes the hitbox?

Reply:
I Used `Button` for examples.